packages/admin/admin/src/alert/Alert.tsx

```diff
@@ -0,0 +1,176 @@
+import { ArrowRight, Check, Close, Error, Info, Warning } from "@comet/admin-icons";
+import { Button, ButtonProps, IconButton, Theme, Typography } from "@mui/material";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import clsx from "clsx";
+import * as React from "react";
+
+type Action = Partial<Omit<ButtonProps, "children">> & {
+    text: ButtonProps["children"];
+};
```

Comment:
Do we really want to allow all Button props here, when we only need `onClick` and `startIcon`? With this approach, one could easily override the `variant` which is set to `"text"` to match the design.

And @jamesricky, wouldn't action be a slot?